spec/core/params.md

```diff
@@ -0,0 +1,224 @@
+|--------------------------------------------------------------|-------|---------------------------------------------------|:------------:|
+| vote_extensions_enable_height / `VoteExtensionsEnableHeight` | int64 | The height where vote extensions will be enabled. | 1            |
+
+The `ABCIParams` type has been **deprecated** from CometBFT `v1.0`.
+
+The vote_extensions_enable_height / `VoteExtensionsEnableHeight` parameter is now part of `FeatureParams`.
+
+### Synchrony
+
+The `SynchronyParams` parameters determine the validity of block timestamps:
+
+| Name                           | Type                                       | Description                                                                                                             | Field Number |
+|--------------------------------|--------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|:------------:|
+| precision / `Precision`        | [google.protobuf.Duration][proto-duration] | Bound for how skewed a proposer's clock may be from any validator on the network while still producing valid proposals. | 1            |
+| message_delay / `MessageDelay` | [google.protobuf.Duration][proto-duration] | Bound for how long a proposal message may take to reach all validators on a network and still be considered valid.      | 2            |
+
+These parameters are part of the Proposer-Based Timestamps (PBTS) algorithm.
+For more information on the relationship of the synchrony parameters to
+block timestamps validity, refer to the [PBTS specification][pbts].
+
+### Feature
+
+The `FeatureParams` parameters configure the height from which features of CometBFT are enabled:
+
+| Name                                                         | Type  | Description                                                       | Field Number |
+|--------------------------------------------------------------|-------|-------------------------------------------------------------------|:------------:|
+| vote_extensions_enable_height / `VoteExtensionsEnableHeight` | int64 | First height during which vote extensions will be enabled.        | 1            |
+| pbts_enable_height / `PbtsEnableHeight`                      | int64 | Height at which Proposer-Based Timestamps (PBTS) will be enabled. | 2            |
+
+If a feature parameter is set to 0, which is the default, then the
+corresponding feature is disabled.
+
+From the configured enable height, and for all subsequent heights, the
+corresponding feature will be enabled.
+
+The feature enable height parameters cannot be set to heights lower or equal to
+the current blockchain height.
+
+Once a feature enable height is set and reached by the blockchain, the feature
+enable height cannot be updated.
+
+## Initial Parameters
+
+TODO: genesis
```

Comment:
It's fine to just put a link to docs.cometbft.com genesis reference here potentially